shoes-core/lib/shoes/dsl.rb

```diff
@@ -308,15 +308,37 @@ def sound(soundfile, opts = {}, &blk)
       Shoes::Sound.new @__app__, soundfile, opts, &blk
     end
 
-    # Creates an arrow centered at (left, top)
+    # Creates an arrow
     #
-    # @param [Integer] left the x-coordinate of the element center
-    # @param [Integer] top the y-coordinate of the element center
-    # @param [Integer] width width of the arrow
-    # @param [Hash] opts Arrow style options
-    # @option opts [Integer] rotate (false)
-    def arrow(left, top, width, styles = {}, &blk)
-      create Shoes::Arrow, left, top, width, styles, blk
+    # @overload arrow(left, top, width, opts)
+    #   Creates an arrow centered at (left, top)
+    #   @param [Integer] left the x-coordinate of the element center
+    #   @param [Integer] top the y-coordinate of the element center
+    #   @param [Integer] width width of the arrow
```

Comment:
Actually not a typo--first word is parameter, everything remaining is description. Could maybe add intervening "the" like other descriptions so it doesn't look mangled though.